source/server/admin/stats_handler.cc

```diff
@@ -85,6 +84,14 @@ Http::Code StatsHandler::handlerStats(absl::string_view url,
     return Http::Code::BadRequest;
   }
 
+  // If the histogram_buckets query param does not exist histogram output should contain quantile
+  // summary data. Using histogram_buckets will change output to show bucket data. The
+  // histogram_buckets query param has two possible values: cumulative or disjoint.
+  Utility::HistogramBucketsValue histogram_buckets_value = Utility::HistogramBucketsValue::Null;
```

Comment:
suggest s/Null/NoBuckets/ or similar? Null seems too generic.

suggest s/_value/_mode/

Reply:
Renamed to NoBuckets and histogram_buckets_mode.